test/unit/account_test.rb

```diff
@@ -361,6 +361,7 @@ def setup
 
     assert_equal "amazing-name-123.#{ThreeScale.config.superdomain}", account.domain
     assert_equal "amazing-name-123-admin.#{ThreeScale.config.superdomain}", account.internal_admin_domain
+    assert_equal "amazing-name-123-admin.#{ThreeScale.config.superdomain}", account.external_admin_domain
```

Comment:
why are these the same? Honest question.

Reply:
https://github.com/3scale/porta/pull/3001#discussion_r912935206